### PR TITLE
AUPR changes in the training script

### PR DESCRIPTION
**Pull request by jzhang-gp**
Change the metric calculation to AUPR based on the discussion on Slack

### CODE REVIEW

File: alchemy/train/no_deps/metrics.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from sklearn.metrics import (
     confusion_matrix,
     precision_recall_fscore_support,
-    roc_auc_score,
+    average_precision_score,
```
Comment (MJafarMashhadi) [inline alchemy/train/no_deps/metrics.py:5]: Maybe we can have a dynamic way to add/remove/update the metrics as a future feature. sklearn has already done much of the heavy lifting on our behalf, i.e. creating a unified interface for all the metrics, we just need to do some basic `importlib` magic to load the metrics dynamically and apply them on the data.

**Review by MJafarMashhadi — APPROVED** [review]

**Comment by angeliney** [comment]
@jzhang-gp can we add the AUPR and still keep the previous ROC AUC?

**Review by angeliney — CHANGES_REQUESTED** [review]
As mentioned in an earlier comment, let's keep the original roc_auc_score but add the average_precision_score on top :) Is that possible?

**Comment by jzhang-gp** [comment]
> @jzhang-gp can we add the AUPR and still keep the previous ROC AUC?

You mean add it as a separate metric? 

> @jzhang-gp can we add the AUPR and still keep the previous ROC AUC?



> As mentioned in an earlier comment, let's keep the original roc_auc_score but add the average_precision_score on top :) Is that possible?

Sure. We can do that.

**Comment by jzhang-gp** [comment]
image
@angeliney does this look OK to you?

**Review by angeliney — APPROVED** [review]
Looks good!!
